packages/m2-theme/public/index.php

```diff
@@ -5,6 +5,7 @@
 $description = $this->getThemeConfiguration('design/head/default_description');
 $themeColor = $this->getThemeConfiguration('webmanifest_customization/webmanifest/theme_color');
 $layoutDirection = $this->getThemeConfiguration('layout_direction_configuration/layout_direction_section/layout_direction') ?: 'ltr';
+$faviconPath = "/media/favicon/" . $this->getThemeConfiguration('design/head/shortcut_icon') ?: '/media/favicon/favicon.png';
```

Comment:
Actually, I assume here will be "/media/favicon/" in return if no favicon provided.